### ISSUE TITLE
PyAutoRelease not working

### ISSUE THREAD

**Issue by TheUntraceable** [EpikCord/EpikCord.py#93]
### Summary

The workflow always crashes.

### Reproduction Steps

Push a commit

### Expected Results

For a new release to be deploying on PyPi

### Actual Results

It errors.

### Intents

Not an issue with Intents.

### System Information

Irrelevant.

### Checklist

- [X] This bug was found on the latest version of EpikCord
- [X] You have checked that there is no issue already mentioning this bug, open or closed
- [X] Full details of the bug have been provided/shown, including full tracebacks in case of exceptions
- [X] All bot tokens have been hidden/omitted from any screenshots or code, if any

**Comment by nav-github01001** [comment]
Bruh 
My commit

**Comment by TheUntraceable** [comment]
It uses EpikHost for the user for some reason, I need to change that but I can't see where it's getting that from?

**Comment by itsnotrin** [comment]
It gets it from the repo secrets / org secrets, no?

**Comment by TheUntraceable** [comment]
No. It doesn't get the *user* from repo secrets but it *does* get the Password/API Key from there

**Comment by Jerry-py** [comment]
Fixed?

**Comment by TheUntraceable** [comment]
Last time I checked, no.

**Comment by EmreTech** [comment]
@TheUntraceable why are you being so vague in this issue? You didn't even provide a traceback from the workflow. You just said "the workflow errors". Not helpful for outsiders at all.
